.prettierignore

```diff
@@ -0,0 +1,46 @@
+# Prettier ignore config - https://prettier.io/docs/en/ignore.html
+
+# This model acts to disallow by default and then enable for specific file types; then exclude within those file types selectively, e.g. include our css, don't include third-party css
+
+# disallow everything, even extensionless files
+*
+# allow directories so the below rules can take effect
+!*/
+
+
+### Allowed File Types ###
+
+# javascript
+!*.jsx
+!*.js
+!*.json
+
+
+# css/sass
+!*.scss
+!*.css
+
+# html
+!*.html
+!*.htm 
```

Comment:
It looks like prettier doesn't know how to handle embedded django template directives, so let's omit html files for now.

![image](https://user-images.githubusercontent.com/322346/106035388-ec3a6200-6088-11eb-9836-45449540dff8.png)

Reply:
Should be resolved now. I updated the prettierignore and then reverted all HTML file changes to current master, 053d576.